lib/spawn-panel.js

```diff
@@ -0,0 +1,15 @@
+const spawn = require("cross-spawn");
+const mv = require("multiview")();
+const path = require("path");
+
+module.exports = async function run(projDir) {
+  let parcelPath = require.resolve(".bin/parcel");
+  let indexPath = path.join(projDir, "app/index.html");
+
+  let parcelProc = mv.spawn(parcelPath, [indexPath]);
+  let npmProc = mv.spawn("npm", ["install"], { cwd: projDir });
+
+  return new Promise((resolve, reject) => {
```

Comment:
reject not used

Reply:
Dang, this file shouldn't even exist anymore.